app/controllers/application_controller.rb

```diff
@@ -8,6 +8,11 @@ class ApplicationController < ActionController::Base
 
   private
 
+  def kudosbot
+    # Initialize kudosbot.
+    @kudosbot ||= Slack::Web::Client.new
```

Comment:
It would be good to have an explicit failure state here... i.e. what happens if (a) a bad Slack API token is passed in [or the token has been revoked] or (b) Slack is offline? This function can throw an exception then whoever calls `kudosbot` can catch it and respond appropriately.